css/properties/scroll-snap-type.json

```diff
@@ -0,0 +1,56 @@
+{
+  "css": {
+    "properties": {
+      "scroll-snap-type": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/scroll-snap-type",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "39"
+            },
+            "firefox_android": {
+              "version_added": "39",
+              "flag": {
+                "type": "preference",
+                "name": "layout.css.vertical-text.enabled",
+                "value_to_set": "true"
+              }
+            },
+            "ie_mobile": {
```

Comment:
edge/edge_mobile are missing. According to https://developer.microsoft.com/en-us/microsoft-edge/platform/status/cssscrollsnappoints/?q=snap
THey are prefixed (like w/ie), so "12", with the -ms prefix.